modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/NavigationPanel.js

```diff
@@ -38,11 +48,49 @@ export default function NavigationPanel({items, selectedItemId}) {
 		navigate(item.href);
 	};
 
+	const handleItemMove = (item, parentItem) => {
+		if (
+			item.type === ITEM_TYPES.folder &&
+			parentItem.type === ITEM_TYPES.article
+		) {
+			return false;
+		}
+
+		fetch(moveKBObjectURL, {
+			body: objectToFormData({
+				[`${portletNamespace}dragAndDrop`]: true,
+				[`${portletNamespace}parentResourceClassNameId`]: parentItem.classNameId,
```

Comment:
can we order this parameters as in

https://github.com/liferay/liferay-portal/blob/b8ccd82fed17ed0ca7f746c452b5f4fcf2a31598/modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_object.jsp#L76

??